frontend/app/views/fragments/promos/promoPrimary.scala.html

```diff
@@ -14,7 +14,7 @@ <h2 class="promo-primary__title h-promo@if(isLead){ h-promo--primary}">@title</h
     </div>
     <div class="promo-primary__media@if(stampImageOpt){ stamped-image}">
         <div class="promo-primary__image">
-            <img src="@Asset.at(imageUrl)" alt="@title" class="responsive-img">
+            <div class="js-imager-loaded-image" data-src="@imageBasePath/{width}.jpg" data-available-widths="1000,500,140" data-alt="@title" data-class="responsive-img"></div>
         </div>
```

Comment:
potential to either 
- Change the current fragments.event.image() fragment to accept three arguments? 
- Or if more similar fragments are required when more promo images are added to the Grid then abstract this into another fragment?